redi-connect-front/src/components/organisms/RediHeroProgram.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import classnames from 'classnames'
+import { Section, Container, Element, Heading, Content, Columns } from 'react-bulma-components'
+import { useTranslation } from 'react-i18next'
+import SVGImage from '../atoms/SVGImage'
+
+import './RediHeroProgram.scss'
+
+interface Props {
+  type: 'mentor' | 'mentee'
+}
+
+const programIcons = [
+  {
+    name: 'career'
+  },
+  {
+    name: 'clipboard'
+  },
+  {
+    name: 'search'
+  },
+  {
+    name: 'calendar'
+  },
+  {
+    name: 'certificate'
+  }
+]
+
+const RediHeroProgram = ({ type }: Props) => {
+  const { t } = useTranslation()
+  const typeSpecificClass = (className: string) => (classnames(`redi-hero__${className}`, {
+    [`redi-hero__${className}--${type}`]: type
+  }))
+
+  return (
+    <>
+      <Section className={classnames({ [`redi-hero__background--${type}`]: type })}>
```

Comment:
Block class is missing here. Just name you component first (`redi-user-hero`) and give it a modifier which will decide this type it will have (`redi-user-hero--mentee`). Most of the type specific classes should be removed. 

https://en.bem.info/methodology/

Reply:
edited